Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.9/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.9/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/Cellar/python@3.9/3.9.7/Frameworks/Python.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/local/lib/python3.9/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
def redaccion_principal(pagina):
    parrafos = obtener_lista_de_parrafos(pagina)
    texto_articulo = ''
    for parrafo in parrafos:
        texto_articulo = texto_articulo + parrafo.text + '\n\n'
    return texto_articulo.strip()
    
redaccion_principal = redaccion_principal(pagina)
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
[0;32m/var/folders/qh/pvjjyjr96l329fgsgrwn11nr0000gp/T/ipykernel_18641/2555584493.py[0m in [0;36m<module>[0;34m[0m
[1;32m      6[0m     [0;32mreturn[0m [0mtexto_articulo[0m[0;34m.[0m[0mstrip[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      7[0m [0;34m[0m[0m
[0;32m----> 8[0;31m [0mredaccion_principal[0m [0;34m=[0m [0mredaccion_principal[0m[0;34m([0m[0mpagina[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/var/folders/qh/pvjjyjr96l329fgsgrwn11nr0000gp/T/ipykernel_18641/2555584493.py[0m in [0;36mredaccion_principal[0;34m(pagina)[0m
[1;32m      1[0m [0;32mdef[0m [0mredaccion_principal[0m[0;34m([0m[0mpagina[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m     [0mparrafos[0m [0;34m=[0m [0mobtener_lista_de_parrafos[0m[0;34m([0m[0mpagina[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m     [0mtexto_articulo[0m [0;34m=[0m [0;34m''[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m     [0;32mfor[0m [0mparrafo[0m [0;32min[0m [0mparrafos[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m         [0mtexto_articulo[0m [0;34m=[0m [0mtexto_articulo[0m [0;34m+[0m [0mparrafo[0m[0;34m.[0m[0mtext[0m [0;34m+[0m [0;34m'\n\n'[0m[0;34m[0m[0;34m[0m[0m

[0;31mNameError[0m: name 'obtener_lista_de_parrafos' is not defined
NameError: name 'obtener_lista_de_parrafos' is not defined

